mailpoet/assets/js/src/automation/api.tsx

```diff
@@ -63,7 +63,7 @@ export const useQuery = <T extends Data>(path: string, init?: RequestInit): Stat
 
   useEffect(
     () => {
-      void mutation();
+      void mutation(); // eslint-disable-line @typescript-eslint/no-floating-promises
```

Comment:
This is strange. The `void` should avoid that error coming from `// eslint-disable-line @typescript-eslint/no-floating-promises`, that's why I allowed void [here](https://github.com/mailpoet/mailpoet/commit/8405659d070f110f49c55e44f84d45c56e4de703).

Reply:
Worked. Not sure why it didn't before. [364b994](https://github.com/mailpoet/mailpoet/pull/3968/commits/364b994295e38917a7e0a01bdb855664dab0742a)